java/api/src/main/java/org/ray/api/id/TaskId.java

```diff
@@ -27,6 +31,25 @@ public static TaskId fromByteBuffer(ByteBuffer bb) {
     return new TaskId(byteBuffer2Bytes(bb));
   }
 
+  public static TaskId generateTaskId(ActorId actorId) {
```

Comment:
It'd be clearer to separate this function into 3: `forActorCreationTask(actorId)`, `forActorTask(actorId)`, and `forNormalTask`.
And for actor creation tasks, the random part can be 0xff. (Please also update the spec doc about this.)
Also, the C++ `TaskId::FromRandom` should be separated as well.

Reply:
Nice idea!